src/encoded/static/components/genome_browser.js

```diff
@@ -445,7 +446,9 @@ class GenomeBrowser extends React.Component {
 
     handleOnFocus() {
         this.setState({ showAutoSuggest: false });
-        getCoordinateData(`${this.context.location_href.split('/experiments/')[0]}/suggest/?genome=${this.state.genome}&q=${this.state.searchTerm}`, this.context.fetch).then((response) => {
+        const urlComponents = url.parse(this.context.location_href);
+        const coordinateHref = `${urlComponents.protocol}//${urlComponents.host}/suggest/?genome=${this.state.genome}&q=${this.state.searchTerm}`;
```

Comment:
Is there a reason you are making an absolute URL rather than an relative one?

Reply:
Yes. If we're making sure this is generic, then we can't be certain what the url is, and need to account for how there could be more parameters than just /experiments/ENCDXXXX or /annotations/ENCSRXXXX. For instance, I'm not sure what the cart page browser url will be, but I do know that we want to use the root of that page to construct this query. So if we were to try to construct a relative url in that case, we would need to know how far up the directory structure to travel. It's simpler, and more robust, in my opinion, to just specify the full url.

Reply:
Ok, tried this and it seems to work! Hopefully it also works on Forrest's cart page. He says he hasn't tried that yet.